examples/flutter_gallery/lib/demo/material/banner_demo.dart

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../gallery/demo.dart';
+
+enum BannerDemoAction {
+  reset,
+  showMultipleActions,
+  showLeading,
+}
+
+class BannerDemo extends StatefulWidget {
+  const BannerDemo({ Key key }) : super(key: key);
+
+  static const String routeName = '/material/banner';
+
+  @override
+  _BannerDemoState createState() => _BannerDemoState();
+}
+
+class _BannerDemoState extends State<BannerDemo> {
+  bool displayBanner = true;
+  bool showMultipleActions = true;
+  bool showLeading = true;
+
+  @override
+  void initState() {
```

Comment:
Since you initialize the three `bool`s at declaration, I don't think you need to override `initState()`.

Reply:
done